.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
     - docker pull ${OS_TYPE}
 
 script:
+    - find \( -name \*.c -o -name \*.h \) -exec sh -c '! grep --color "^ *//" $@' find {} \+ && echo No line comments found...
```

Comment:
The grep won't catch comments indented by tabs because AST code uses tabs, not spaces.
Perhaps this: ! grep --color "^[[:space:]]*//" $@'
But that brings up another point.  Should we be also checking that tabs have been used?